plugin/src/stores/remixClient.ts

```diff
@@ -8,7 +8,12 @@ const WORKSPACE_ROOT = '.workspaces'
 const remixClientAtom = atom({} as unknown as RemixClient)
 const noFileSelectedAtom = atom(false)
 const isValidSolidityAtom = atom(false)
-const currentFilenameAtom = atom('')
+const currentFilepathAtom = atom('')
+const currentFilenameAtom = atom((get) => {
```

Comment:
@satyambnsal There's no really a need in an atom for this. When it's used we can explicitly call `getFileNameFromPath` on `currentFilepathAtom`. Right now when reading `Compilation.txs` there's impression that those 2 can have different states.

Reply:
Its a derived read only atom where the atom value gets derived from `currentFilepathAtom`.

Reply:
Yeah, I saw that, but I don't think there's an atom needed for that since the only atom we need is `filePath`